examples/ciao/README.md

```diff
@@ -5,12 +5,18 @@ This is an example of a playbook to deploy CIAO using ansible.
 ## Prework
 
 ### Access
-Ansible requires that the user running the playbook has passwordless ssh access to the managed nodes, sudo privileges and python installed on each node.
+Ansible requires that the user running the playbook has passwordless ssh access
```

Comment:
You need passwordless ssh from the deployment machine to all the managed nodes. Not between managed nodes.